### PR TITLE
docs: additional capabilities pages DOC-1144

### PR DESCRIPTION
**Pull request by addetz**
## Describe the Change

This PR breaks out the sections of the additional capabilities page into their own pages. It also adds relevant resources sections to each page.

## Changed Pages

💻 [Additional Capabilities](https://deploy-preview-3550--docs-spectrocloud.netlify.app/getting-started/additional-capabilities/)
💻 [Edge](https://deploy-preview-3550--docs-spectrocloud.netlify.app/getting-started/additional-capabilities/edge)
💻 [VerteX](https://deploy-preview-3550--docs-spectrocloud.netlify.app/getting-started/additional-capabilities/self-hosted)
💻 [VMO](https://deploy-preview-3550--docs-spectrocloud.netlify.app/getting-started/additional-capabilities/vmo)

## Jira Tickets

🎫 [DOC-1144](https://spectrocloud.atlassian.net/browse/DOC-1144)

## Backports

Can this PR be backported?

- [ ] Yes. _Remember to add the relevant backport labels to your PR._
- [x] No. _Please leave a short comment below about why this PR cannot be backported._

[DOC-1144]: https://spectrocloud.atlassian.net/browse/DOC-1144?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/getting-started/additional-capabilities/vmo.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+hypervisor layer. This is achieved by leveraging [Canonical MAAS](https://maas.io). Additionally, VMO can also be used
+in self-hosted, airgapped, and in our SaaS environments. Learn more on the
+[Virtual Machine Orchestrator](../../vm-management/vm-management.md) page.
+
+## Resources
+
+To learn more about Palette VMO, review the [Architecture](../../vm-management/architecture.md) page to learn about the
+components involved in enabling VMO for your infrastructure. Then, review the
+[Create a VMO Profile](../../vm-management/create-vmo-profile.md) guide to prepare everything you need to deploy your
+first VMO cluster.
```
Comment (caroldelwing) [inline docs/docs-content/getting-started/additional-capabilities/vmo.md:29]:
```suggestion
first cluster with VMO.
```

File: docs/docs-content/getting-started/additional-capabilities/vmo.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Check out the following video for a tour of Palette's Virtual Machine Orchestrator (VMO) capability, showing how you can
```
Comment (caroldelwing) [inline docs/docs-content/getting-started/additional-capabilities/vmo.md:31]:
```suggestion
Check out the following video for a tour of Palette's Virtual Machine Orchestrator (VMO) capability. It shows how you can
```

File: docs/docs-content/getting-started/additional-capabilities/vmo.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+model, deploy and manage VM workloads alongside containers in all your clusters and environments.
```
Comment (caroldelwing) [inline docs/docs-content/getting-started/additional-capabilities/vmo.md:32]:
```suggestion
model, deploy, and manage VM workloads alongside containers in your clusters.
```
maybe?

**Review by caroldelwing — APPROVED** [review]
Just tiny suggestions 🚀

File: docs/docs-content/getting-started/additional-capabilities/self-hosted.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+Review the [Self-Hosted Palette](../../enterprise-version/enterprise-version.md) section to learn how to install the
+self-hosted version of Palette in your data centers or public cloud providers to manage Kubernetes clusters.
+
+Check out the following video to discover some simple strategies you can follow to streamline how you manage Kubernetes
+deployments and versioning in your production environment.
+
+<br />
+
+<YouTube
```
[the remark this answers is not in the document]
Comment (addetz) [reply docs/docs-content/getting-started/additional-capabilities/self-hosted.md:33]: I had a hard time finding a good vide for this one. I removed it and put the self hosted link in a single resources section at the bottom.

**Pull request merged**